### DISCUSSION TITLE
gopro 11, resolve 18.5, aces

### DISCUSSION THREAD

**Discussion by p0las** [xtremestuff/protune-transforms#8] [Q&A]
not sure what is going on yet but once I apply the input transform the footage gets very dark and yellow. nothing that I cannot correct later but it suggests that I'm not using it right. I shot with native white balance one stop underexposed. applied the input transform to the clip directly with my timeline set to sRGB.

**Comment by RikshaDriver** [comment]
What are your project color settings?

Native does not include any white-balancing, hence the yellow appearance.

If you've under-exposed deliberately, then your footage would likely appear dark without the appropriate exposure adjustement. The Protune curve has a mid-gray point that is very high at 68% IRE, so anything below that will be darker when corrected.

**Reply by p0las** [reply]
ok. great. so it appears that this is a desired base look that needs to be graded. it grades nicely so I have no issue with this approach. Just wanted to make sure I'm not doing extra work that could have been avoided :-)

my project settings are aces cc, no input transform with standard sRGB output one.

the results are surprisingly good for a gopro :-). I was able to match it effortlessly to a v-log footage from my panasonic. I didn't stress test the dynamic range but colour wise the results are nice.
https://youtu.be/EIApm5hRDpw

thank you for putting the effort into it.
